pkg/process/checks/process.go

```diff
@@ -18,8 +18,6 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/process/util"
 	"github.com/DataDog/datadog-agent/pkg/util/cloudproviders"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
-	"github.com/DataDog/gopsutil/cpu"
-	"go.uber.org/atomic"
```

Comment:
Looks like this was removed unintentionally?

Reply:
Yes, adding back in.